bcipy/task/base_calibration.py

```diff
@@ -0,0 +1,324 @@
+"""Base calibration task."""
+
+from typing import Any, Dict, Iterator, List, NamedTuple, Optional, Tuple
+
+from psychopy import core, visual
+
+import bcipy.task.data as session_data
+from bcipy.acquisition import ClientManager
+from bcipy.config import (SESSION_DATA_FILENAME, TRIGGER_FILENAME,
+                          WAIT_SCREEN_MESSAGE)
+from bcipy.display import Display
+from bcipy.helpers.clock import Clock
+from bcipy.helpers.parameters import Parameters
+from bcipy.helpers.save import _save_session_related_data
+from bcipy.helpers.stimuli import (DEFAULT_TEXT_FIXATION, StimuliOrder,
+                                   TargetPositions,
+                                   generate_calibration_inquiries)
+from bcipy.helpers.symbols import alphabet
+from bcipy.helpers.task import (get_user_input, pause_calibration,
+                                trial_complete_message)
+from bcipy.helpers.triggers import (FlushFrequency, Trigger, TriggerHandler,
+                                    TriggerType, convert_timing_triggers,
+                                    offset_label)
+from bcipy.task import Task
+
+
+class Inquiry(NamedTuple):
+    """Represents a single Inquiry"""
+    stimuli: List[str]
+    durations: List[float]
+    colors: List[str]
+
+    @property
+    def target(self) -> str:
```

Comment:
Some documentation on the expected stimuli would be helpful. It seems like it would expect `target fixation inquiry_stimuli`?